dashboard/v1.1/src/pages/Input/components/MaterialLabelSelector.tsx

```diff
@@ -43,6 +42,9 @@ export default function MaterialLabelSelector(props: LabelSelectorType) {
       name: labelValue,
       color: stc(labelValue)
     };
+    setPendingValue([...pendingValue, newLabel]);
+    setValue(pendingValue);
+    updateLabels(pendingValue);
```

Comment:
Can you describe in detail how this change improves the LabelSelector in the PR description, would be useful for folks to keep that in mind while reviewing this PR.

Reply:
@aquibbaig I have added functionality that as soon as you create a new label it is applied automatically like in GitHub when we create an issue label it is applied on its own. Plus there was a bug that once you select a created label you cannot undo it but now if you want to remove that label you can do that and still, it will remain there in the label list. In the above code label list, labels applied are being updated as soon as a new label is created.